client/src/locales/ru.ts

```diff
@@ -57,7 +59,8 @@ export default {
 		},
 	},
 	"footer": {
-		"disclaimer": "Дисклеймер: OpenTogetherTube никак не связан с TogetherTube и Watch2Gether.",
+		"disclaimer":
```

Comment:
I don't understand why you removed the line break, because it's not in en.ts

Reply:
probably the auto formatter. its fine as long as it's valid syntax